BrainPortal/app/views/layouts/_neurohub_flash_message.html.erb

```diff
@@ -35,3 +35,16 @@
       <%= (h(msg.to_s.strip).gsub(/\n/, '<BR/>')).html_safe %>
     </div>
   <%- end -%>
+
+  <div class="card">
```

Comment:
Shouldn't the div be inside the `if` block ? Otherwise we get a div with nothing inside.

Reply:
it should be inside the `unless` block.

Reply:
+ same comment as @prioux made later:
> Change the unless something.blank? to if something.present? it looks much nicer and avoids double negatives.